cdk.json

```diff
@@ -29,6 +29,39 @@
     "@aws-cdk/core:target-partitions": [
       "aws",
       "aws-cn"
-    ]
+    ],
+    "dev": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:0.2.8-beta",
+      "AWS_DEFAULT_REGION": "us-east-1",
+      "PORT": "3838",
+      "COST_CENTER": "NO PROGRAM / 000000",
+      "STACK_NAME_PREFIX": "dca-test",
+      "HOST_NAME": "dca.dnt-dev.sagebase.org",
+      "HOSTED_ZONE_NAME": "dnt-dev.sagebase.org",
+      "HOSTED_ZONE_ID": "Z0441843EAHKE7DR12AX",
+      "VPC_CIDR": "10.255.70.0/24"
+    },
+    "staging": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:0.2.8-beta",
+      "AWS_DEFAULT_REGION": "us-east-1",
+      "PORT": "3838",
+      "COST_CENTER": "NO PROGRAM / 000000",
+      "STACK_NAME_PREFIX": "dca-test",
+      "HOST_NAME": "dca.dnt-dev.sagebase.org",
+      "HOSTED_ZONE_NAME": "dnt-dev.sagebase.org",
+      "HOSTED_ZONE_ID": "Z0441843EAHKE7DR12AX",
+      "VPC_CIDR": "10.255.71.0/24"
+    },
+    "prod": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:0.2.8-beta",
```

Comment:
I assume these values are placeholders, to be updated once we have production resources set up.

Reply:
yes, that is correct.